README.adoc

```diff
@@ -68,17 +62,20 @@ mvn install liberty:start-server
 
 Maven builds the application and runs it inside Open Liberty.
 
-Point your browser to the `http://localhost:9080/inventory/systems` URL.
-Because you just started the application, the inventory is currently empty. Point to the
+Point your browser to the `http://localhost:9080/inventory/systems` URL to access the `inventory`
+service. Because you just started the application, the inventory is currently empty. Then, access the
 `http://localhost:9080/inventory/systems/localhost` URL to add the localhost host into the inventory.
```

Comment:
the localhost host. I just noticed that. should it be `localhost` or just localhost without host after it. ?